models/patch.go

```diff
@@ -0,0 +1,98 @@
+package models
+
+import (
+	"time"
+	"fmt"
+	"log"
+)
+
+type Patch struct {
+	Timestamp   time.Time      		 `json:"timestamp"`
+	Patch 		string 				 `json:"patch"`
+	Convo_ID    string               `json:"convo_id"`
+	User_ID     string         		 `json:"user_id"`
+	Type 		string 				 `json:"type"`
+}
+
+type Convo_ID struct {
+	Convo_ID   int64      		 `json:"convo_id,string"`
+}
+
+// Get patch rows from database using filters
+func (db *DB) GetPatches(filterString string) ([]Patch, error) {
+
+	// Create query string with filters
+	queryString := fmt.Sprintf("%s%s", "SELECT * FROM patches", filterString)
+	rows, err := db.Query(queryString)
+	if err != nil {
+		log.Print("Error getting rows")
+		log.Print(err)
+		return nil, err
+	}
```

Comment:
You should put `defer rows.Close()` here to unsure that the connection gets closed once this function returns.